src/app/chat/chat-room/chat-room.component.html

```diff
@@ -31,10 +36,11 @@
   </ion-list>
 </ion-content>
 <ion-footer>
-    <ion-item lines="none">
-        <ion-textarea placeholder="Enter your message" (keydown.enter)="sendMessage($event)" required="true" name="message" [(ngModel)]="message"></ion-textarea>
-      <ion-button (click)="sendMessage()" type="submit" slot="end" fill="clear" color="light">
-        <ion-icon name="send"></ion-icon>
-      </ion-button>
-    </ion-item>
-</ion-footer>
\ No newline at end of file
+  <ion-item lines="none">
+    <ion-textarea placeholder="Enter your message" (keydown.enter)="sendMessage($event)" required="true" name="message"
```

Comment:
Pressing enter should go to a new line instead of send the message

Reply:
This is the way we implement this logic in app v1 and also this is the way every other chat apps  web version using. Why we need to change this?

Reply:
The reason is that on mobile you don't have "Shift+Enter"/"Command+Enter"

Reply:
we need to ask this from anu and beenish before change.